pkgs/applications/science/logic/opensmt/default.nix

```diff
@@ -1,25 +1,38 @@
-{ stdenv, fetchurl, automake, libtool, autoconf, intltool, perl
-, gmpxx, flex, bison
+{ stdenv, lib, fetchFromGitHub
+, cmake, libedit, gmpxx, bison, flex
+, enableReadline ? false, readline
```

Comment:
Is there any scenario where you want to disable readline support? It could also be just enabled by default to add useless options.

Reply:
Well it impacts the license

Reply:
Someone maybe cares about this.